static/src/javascripts-legacy/projects/common/modules/navigation/search.js

```diff
@@ -63,7 +68,6 @@ define([
                 // Make sure search is always in the correct state
                 self.focusSearchField();
                 e.preventDefault();
-                mediator.emit('modules:search');
```

Comment:
Thanks a good reason! Might be good in future to keep that as a separate commit as it doesn't actually relate to `"Search: prevent closing of the pop-up on autocomplete select"`